src/style.css

```diff
@@ -1,3 +1,92 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  --header-height: 60px;
+  background-color: rgb(209, 207, 207);
+}
+
+.page__body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  height: var(--header-height);
+
+  display: flex;
+  justify-content: flex-end;
+  background-color: white;
+  align-items: center;
+  padding: 0 50px;
+  cursor: pointer;
+}
+
+.logo {
+  margin-right: auto;
+}
+
+.logo img {
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+}
+
+.nav__list {
+  list-style: none;
+}
+
+.nav__item {
+  display: inline-block;
+}
+
+.nav__link {
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-left: 20px;
+  text-align: center;
+  position: relative;
+  z-index: 50;
+}
+
+.nav__link::before {
+  position: absolute;
+  content: " ";
+  top: -20px;
+  right: -20px;
+  left: -20px;
+  bottom: -20px;
+  z-index: 40;
```

Comment:
dont use so huge z-index its can cause for unexpected behavior,
and why do you need this before?